crates/css-syntax/src/syntax.rs

```diff
@@ -821,10 +821,12 @@ fn write_formal_syntax_internal(
     write!(out, r#"<pre class="notranslate css-formal-syntax">"#)?;
     let mut constituents = renderer.get_constituent_syntaxes(syntax)?;
 
+    let mut constituents_out = String::new();
     for constituent in constituents.iter().skip(if skip_first { 1 } else { 0 }) {
-        renderer.render(&mut out, constituent)?;
-        out.push_str("<br/>");
+        renderer.render(&mut constituents_out, constituent)?;
+        constituents_out.push_str("<br/>");
     }
+    out.push_str(constituents_out.trim_end_matches("<br/>").trim_end());
```

Comment:
```rust
    for (i, constituent) in constituents
        .iter()
        .skip(if skip_first { 1 } else { 0 })
        .enumerate()
    {
        if i > 0 {
            out.push_str("<br/>");
        }
        renderer.render(&mut out, constituent)?;
    }
```

This would do the trick but leaves the last `<br/>` but I think keeping it is better.

Reply:
What is the purpose of the trailing `<br/>`?

Reply:
Sorry, there is no purpose. Most of our formatters add a trailing new line it has ups and downs. But I really see no reason why this matters here?

Reply:
Okay, changes applied.